Source/core/Time.cpp

```diff
@@ -779,6 +814,21 @@ namespace Core {
         _ticks = (static_cast<uint64_t>(time.tv_sec) * MicroSecondsPerSecond) + (time.tv_nsec / NanoSecondsPerMicroSecond) + OffsetTicksForEpoch;
     }
 
+    Time Time::ToLocal() const {
+        struct tm local = _time;
+        time_t flatTime;
+        flatTime = mktime(&local);
+        localtime_r(&flatTime, &local);
```

Comment:
Why we are doing broken-down time representation to local time, then again reverse ?

Reply:
To allow a now time (alwyas in UTC) t be returned as a local time (using the locale).
Internally we calculate with UTC times, So any timer that is being set will be done in UTC, If someone now would like to know what that time would be in the system set local time zone, it can be done by the calling sequence:
Core::Time::Now().ToLocal()
The returned time for us (in CET) will than be 1 hour earlier.

Reply:
Ok, thanks